splitio/client/factory.py

```diff
@@ -350,7 +350,7 @@ def _build_in_memory_factory(api_key, cfg, sdk_url=None, events_url=None,  # pyl
     }
 
     storages = {
-        'splits': InMemorySplitStorage(),
+        'splits': InMemorySplitStorage(cfg['FlagSetsFilter'] if cfg['FlagSetsFilter'] is not None else []),
```

Comment:
we should be consistent with cfg and use first letter lowercase

Reply:
will fix